dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/lksg/categories/general/subcategories/production/LksgProductionSpecificOwnOperations.kt

```diff
@@ -0,0 +1,17 @@
+package org.dataland.datalandbackend.model.lksg.categories.general.subcategories.production
+
+import io.swagger.v3.oas.annotations.media.Schema
+import org.dataland.datalandbackend.model.enums.lksg.ProcurementCategory
+import org.dataland.datalandbackend.utils.JsonExampleFormattingConstants
+
+/**
+ * --- API model ---
+ * Fields of the LKSG questionnaire regarding the impact topic "Production Specific"
+ */
+data class LksgProductionSpecificOwnOperations(
+
+    val mostImportantProducts: List<LksgProduct>?,
+
+    @field:Schema(example = JsonExampleFormattingConstants.PRODUCT_CATEGORIES_DEFAULT_VALUE)
```

Comment:
This brought up a lot of questions for me. Maybe we can discuss this.

Reply:
Gerne.

Reply:
Clarified on the phone. Thx again 👍